apps/synapse-portal-framework/src/components/navbar/DropdownNavButton.tsx

```diff
@@ -0,0 +1,137 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Button,
+  Menu,
+  MenuItem,
+  useMediaQuery,
+  useTheme,
+} from '@mui/material'
+import React from 'react'
+import { useMatch } from 'react-router-dom'
+import NavLink from '../NavLink'
+
+export function DropdownNavButton(props) {
```

Comment:
Rewrote Navbar rendering logic, this component handles nested nav routes and now uses MUI:

Desktop:

![image](https://github.com/user-attachments/assets/8a5e0ae5-4a59-40e6-ba04-e32efe808935)

Mobile:

![image](https://github.com/user-attachments/assets/eead0ff6-7a7f-4ba0-8aca-758becc0431a)